design-library/src/main/java/com/what3words/design/library/ui/components/What3wordsAddress.kt

```diff
@@ -70,68 +89,130 @@ object What3wordsAddressDefaults {
     }
 
     /**
-     * Provides default text style settings for What3wordsAddress composable.
+     * Composable function that provides default text style configurations for the What3wordsAddress composable function.
      *
-     * @param wordsTextStyle Default text style for words, defaults to What3words typography headlineLargeProminent style.
-     * @return An instance of [TextStyles] with the specified or default text style settings.
+     * @param wordsTextStyle The text style to be used for words text. Default is set by [MaterialTheme.w3wTypography.headlineLargeProminent].
+     * @return A TextStyles object with the provided text style configurations.
      */
     @Composable
     fun defaultTextStyles(
-        wordsTextStyle: TextStyle = MaterialTheme.w3wTypography.headlineLargeProminent,
+        wordsTextStyle: TextStyle = MaterialTheme.w3wTypography.headlineLargeProminent
     ): TextStyles {
         return TextStyles(
             wordsTextStyle = wordsTextStyle
         )
     }
 }
 
+/**
+ * A composable function that displays a What3words address with a secondary text.
+ *
+ * @param words The string of words to be displayed.
+ * @param modifier The Modifier to be applied to the layout.
+ * @param colors The Colors object for color configurations. Default is set by [What3wordsAddressDefaults.defaultColors].
+ * @param textStyles The TextStyles object for text style configurations. Default is set by [What3wordsAddressDefaults.defaultTextStyles].
+ * @param secondaryText The secondary text string to be displayed.
+ * @param slashesMargin A lambda function for setting the margin of slashes.
+ */
+@Composable
+fun What3wordsAddress(
+    words: String,
+    modifier: Modifier = Modifier,
+    colors: What3wordsAddressDefaults.Colors = What3wordsAddressDefaults.defaultColors(),
+    textStyles: What3wordsAddressDefaults.TextStyles = What3wordsAddressDefaults.defaultTextStyles(),
+    secondaryText: String,
```

Comment:
Hi @mani-w3w, Should we expose the TexStyle of the secondaryText that helps us scale up the font size if need

Reply:
Good point I will add that 👍

Reply:
done @kaytran2992

Reply:
LTGM